tests/models/test_models.py

```diff
@@ -366,6 +366,84 @@ def test_partial_patch_trial_metadata(clean_db):
     TrialMetadata._patch_trial_metadata(TRIAL_ID, metadata_patch)
 
 
+@db_test
+def test_trial_metadata_get_summaries(clean_db, monkeypatch):
+    """Check that trial data summaries are computed as expected"""
+    monkeypatch.setattr(
+        "cidc_api.models.models.DISABLE_METADATA_VALIDATION_ON_INSERT", True
+    )
```

Comment:
Based on what I tried before taking the hacky approach, no - maybe it was user error on my part, but it may also be because of how sqlalchemy's `@validates` decorator works. It's possible that by looking into sqlalchemy implementation internals I'll be able to find what needs to be mocked, though.

Reply:
one way to disentangle from `@validates` is to introduce one extra indirection layer, something like:
```
@validates
def validate_metadata_json_on_insert(...):
    return _acutally_validate_mdjson(...)

def _acutally_validate_mdjson(...)
```
and then monkeypatch `_acutally_validate_mdjson`

Reply:
Oh, of course! Should've thought of that 😅 Thanks @curlup.